ros2topic/ros2topic/verb/pub.py

```diff
@@ -83,6 +83,10 @@ def add_arguments(self, parser, cli_name):
         group.add_argument(
             '-t', '--times', type=nonnegative_int, default=0,
             help='Publish this number of times and then exit')
+        parser.add_argument(
+            '--keep-alive', metavar='N', type=positive_float, default=0.1,
+            help='Keep publishing node alive for N seconds after the last msg '
+                 '(default: 0.1)')
```

Comment:
@dirk-thomas meta, unrelated: I thought the parser took care of adding a string representation of the given `default` to the `help` string. But it does not. I think it should.

Reply:
We are not configuring `argparse` for that. We are use the raw formatter class in `ros2cli`: https://github.com/ros2/ros2cli/blob/673d4fd22c17e59377f30005a3f82a43e425ecda/ros2cli/ros2cli/cli.py#L30